scripts/mods/mod_dynamic_perks/classes/perk_tree.nut

```diff
@@ -232,6 +232,17 @@ this.perk_tree <- ::inherit(::MSU.BBClass.Empty, {
 		return ret;
 	}
 
+	function getPerkGroupID(_perkID)
+	{
+		foreach (perkGroupID, perkGroup in ::DynamicPerks.PerkGroups.getAll())
+		{
+			if (this.hasPerkGroup(perkGroupID) && perkGroup.hasPerk(_perkID)) {
+				return perkGroupID;
+			}
+		}
+		return "";
```

Comment:
Not sure how I feel about returning an empty string here, this should be a `null` that needs to be separate handled imo

Reply:
I don't think this is really the place that should care if there's a missing perk group. That should become a concern much earlier. Anyways, I didn't know if perks can just have no perk group, so I figured just an empty string was the sensible default.